src/pages/networks/forms/NetworkParentSelector.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+import { Select } from "@canonical/react-components";
+import Loader from "components/Loader";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchNetworks } from "api/networks";
+
+interface Props {
+  isDisabled: boolean;
+  props?: Record<string, unknown>;
+}
+
+const NetworkParentSelector: FC<Props> = ({ isDisabled, props }) => {
+  const { data: networks = [], isLoading: isNetworkLoading } = useQuery({
+    queryKey: [queryKeys.networks, "default"],
+    queryFn: () => fetchNetworks("default"),
```

Comment:
does all existing interfaces come from the `default` project?

Reply:
Good catch, we should use the current project here instead.